CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ChangeLog
 ---------
 
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
```

Comment:
IMHO a change from the MIT to Apache Licence would probably have been worth a note in the ChangeLog?